.kokoro/lint.sh

```diff
@@ -23,11 +23,11 @@ cd $(dirname $0)/..
 npm install
 
 # Install and link samples
-if [ -f samples/package.json ]; then
-  cd samples/
-  npm link ../
-  npm install
-  cd ..
-fi
+#if [ -f samples/package.json ]; then
```

Comment:
I'd rather see a skeleton quickstart and sample test than commenting these out

Reply:
there's a fully functional `quickstart.js` test; as I understand it though, it won't work until we publish the first version of this library to npm (since the quickstart tests perform an `npm install` of the module).